requirements/base.txt

```diff
@@ -21,6 +21,12 @@ pyopenssl==22.0.0
 
 # Needed for supporting Protocol in Python 3.7, Protocol class became public with python3.8
 typing_extensions==3.10.0.0
+# Typeshed type stubs
+types-pywin32==304.0.0.1
+types-PyYAML==6.0.12
```

Comment:
Why are we adding these to `base.txt` instead of `dev.txt`? We shouldn't need these outside of development right?